crates/sui-transaction-builder/src/lib.rs

```diff
@@ -524,7 +530,7 @@ impl<Mode: ExecutionMode> TransactionBuilder<Mode> {
         ))
     }
 
-    pub async fn batch_transaction(
+    pub async fn batch_transaction<Mode: ExecutionMode>(
```

Comment:
Don't we only support `ExecutionMode::Normal` for batch transactions?

Reply:
We support batch in dev inspect IIRC